### PR TITLE
TDRD-496 - Update draft metadata test to validate and verify error report

### PR DESCRIPTION
**Pull request by vimleshtna**

### CODE REVIEW

File: src/test/scala/steps/Steps.scala [hunk 1]
```diff
@@ -729,6 +740,19 @@ class Steps extends ScalaDsl with EN {
       Assert.assertTrue(expectedRows.containsSlice(actualRows))
   }
 
+  Then("^the error report should be same as (.*)") {
+    fileName: String =>
+      val metadataCsv = getDownloadedCsv("ErrorReport").last
+      val wb = new ReadableWorkbook(metadataCsv)
+      val sheet = wb.getFirstSheet
+      val actualRows = sheet.read().asScala.map(_.asScala.map(_.getText).mkString(",")).toList
+      println(actualRows.mkString("\n"))
```
Comment (Tom-Hallett) [inline src/test/scala/steps/Steps.scala:749]: Is this just a debug line or required?

File: build.sbt [hunk 2]
```diff
@@ -14,6 +14,7 @@ libraryDependencies ++= Seq(
   tdrGenerateGraphQl,
   circeCore,
   circeGeneric,
+  fastexceReader,
```
Comment (Tom-Hallett) [inline build.sbt:17]: Rename to fastexcelReader (and below)

**Review by Tom-Hallett — CHANGES_REQUESTED** [review]
Some minor comments

File: src/test/scala/steps/Steps.scala (hunk 1 above)
Comment (vimleshtna) [reply src/test/scala/steps/Steps.scala]: Removed it. Thanks

**Review by Tom-Hallett — APPROVED** [review]
LGTM - thanks

**Pull request merged**